### PR TITLE
[V2PROD-613] Build the initial featureset of Bloom API

### PR DESCRIPTION
**Pull request by ethereumdegen**

### CODE REVIEW

File: server/controllers/api-controller.ts [hunk 1]
```diff
@@ -1,40 +1,68 @@
...
-    console.log('received webhook')
+    try {
+      createdRecord = await this.mongoInterface.WebhookReceiptModel.create(
+        receipt
+      )
+
+      console.log('inserted', createdRecord)
+    } catch (error) {
+      console.error(error)
+    }
```
Comment (passabilities) [inline server/controllers/api-controller.ts]: I think we should add better error checking here. rn if there is an error when inserting a record in the db, the webhook event data will essentially be forgotten about, other than an email being sent. 

are there any mongo errors that we need to be concerned with that could result in loss of data?

File: server/controllers/api-controller.ts [hunk 2]
```diff
@@ -1,40 +1,68 @@
...
+
+    try {
+      sentEmail = await sendEmail(
+        'Bloom API Alert',
+        'A webhook has been received with request_id '.concat(inputs.requestId)
+      )
```
Comment (passabilities) [inline server/controllers/api-controller.ts:63]: Seems like not a lot of data in the email. I assume @udaiparvatha is ok with this?

File: server/server.ts [hunk 3]
```diff
@@ -3,51 +3,50 @@ import http from 'http'
...
+    
+    app.use(
+      bodyParser.json({
+        type: '*/*',
+        verify: (req:any, res:any, buf:Buffer) => {
+           req.rawBody = buf
+           return true
+        },
+        limit: '10mb', // https://stackoverflow.com/a/19965089/1165441
       })
```
Comment (passabilities) [inline server/server.ts:48]: hmm curious what this does?

File: yarn-error.log [hunk 4]
```diff
@@ -0,0 +1,4980 @@
+Arguments:
```
Comment (passabilities) [inline yarn-error.log]: please add this to the `.gitignore`:
1. delete the file
2. commit the change
3. add to `.gitignore`
4. commit the changes

**Review by passabilities — CHANGES_REQUESTED** [review]

**Review by passabilities — APPROVED** [review]

File: server/controllers/api-controller.ts (hunk 1 above)
Comment (ethereumdegen) [reply server/controllers/api-controller.ts]: added an error check for missing request id .  The mongo db does do type checking all built in (see mongo-interface.ts)

File: server/controllers/api-controller.ts (hunk 2 above)
Comment (ethereumdegen) [reply server/controllers/api-controller.ts:63]: Yes Udai is okay with this, i showed him how to look up the data in mongo for now

File: server/server.ts (hunk 3 above)
Comment (ethereumdegen) [reply server/server.ts:48]: I added a comment about this.  It is necessary as per the bloom devs due to how they send their webhook data

**Pull request merged**